src/cf.data.pre

```diff
@@ -1905,12 +1914,6 @@ http_access deny to_linklocal
 # INSERT YOUR OWN RULE(S) HERE TO ALLOW ACCESS FROM YOUR CLIENTS
 #
 
-# Example rule allowing access from your local networks.
-# Adapt localnet in the ACL section to list your (internal) IP networks
-# from where browsing should be allowed
-http_access allow localnet
```

Comment:
Please only comment out this localnet line and its comment above, instead of removing fully.

Having them here documents for beginners how to use localnet properly so they do not accidentally create an open-proxy by adding it earlier where localnet is defined.

Reply:
Done at 689fc25. I rephrased to remove the suggestion to modify the `localnet` and get rid of potentially dangerous/misleading browsing-based verbiage. As you know, I think we should make most if not all default-defined ACLs constant.

If you insist, I will restore the original wording because this change can be considered outside this PR scope.